tasks/install.yml

```diff
@@ -1,4 +1,22 @@
 ---
+- name: Add EPEL repository
+  package:
+    name: epel-release
+    state: latest
+  tags:
+    - skip_ansible_lint
+
+# Around 2017-07-12, installing epel-release stopped being idempotent. The
```

Comment:
It looks like they didn't bother to upload the epel-release 7-10 package to the CentOS Extras repository.

This is what changed between those packages:

```
-mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
+metalink=https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
```

Another way to enable it would be to use the yum_repository module:

```yaml
yum_repository:
  name: epel
  description: Extra Packages for Enterprise Linux 7 - $basearch
  metalink: https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
  failovermethod: priority
  enabled: yes
  gpgkey: http://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
  gpgcheck: yes
```